test/OpenTelemetry.Tests/Trace/ExportProcessorTest.cs

```diff
@@ -26,12 +25,13 @@ namespace OpenTelemetry.Tests.Trace
     public class ExportProcessorTest
     {
         private const string ActivitySourceName = "ActivityExportProcessorTest";
+        private List<Activity> exportedActivities = new List<Activity>();
 
         [Fact]
         public void ExportProcessorIgnoresActivityWhenDropped()
         {
             var sampler = new AlwaysOffSampler();
-            var processor = new TestActivityExportProcessor(new ConsoleActivityExporter(null));
+            var processor = new TestActivityExportProcessor(new InMemoryExporter<Activity>(this.exportedActivities));
```

Comment:
As this test is not interested in the actual values, you can just pass null here.

Reply:
after reading again, since this test is checking if Processor is called or not, use any Processor. It need not be an exporting processor.

Reply:
These tests are checking if the Activity.Recorded flag is being used to determine if the activity needs to be exported.  The Processor OnEnd will be called anyway. Therefore, it tests that there no items added to the collection passed.  That's why I used an ExportProcessor.

Reply:
Got it. That also means - you don't need InMemoryExporter here right? it can be dummy exporter/noop one. (like the TestExporter)

Reply:
Yeah we just have to pass some exporter and TestExporter with a noop action seems best for this. I have updated it to use the TestExporter.

Reply:
thnx. Its a lot cleaner now.